### ISSUE TITLE
Many Issues Submitted to Errata Page

### ISSUE THREAD

**Issue by nobozo** [lorin/ansiblebook#12]
Lorin,

Sorry to abuse the Github new issue function but I've submitted a whole bunch of issues to the O'Reilly errata page for this book. It doesn't appear you pay much attention to that page. Could you please take a look at it. (I don't use Twitter or other social networks so I can't contact you using them.)

**Comment by lorin** [comment]
Jon:

Sorry about that... It looks like I haven't been receiving email notifications about new errata. I'll take a look at them.

**Comment by nobozo** [comment]
On 11/20/16 12:31 PM, Lorin Hochstein wrote:

> Jon:
> 
> Sorry about that... It looks like I haven't been receiving email
> notifications about new errata. I'll take a look at them.

I'm glad I was able to reach you.

As you'll see, I've found a large number of non-technical typos
and consistency issues, plus a few actual mistakes.

I'm reading your book because I have to come up to speed on Ansible
in a hurry. So far I'm enjoying it a lot, although there are some
areas where it could be made clearer.

Cordially,
Jon Forrest

P.S. If you'd like to send me your private email I can send
future comments to you directly. I'll continue to submit actual
book issues in the Errata.
